### PR TITLE
OCPBUGS-23201-4.10: fixes RHEL version bump on 4.10

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.10

Issue:
https://issues.redhat.com/browse/OCPBUGS-23201

Link to docs preview:
https://68102--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_platform_agnostic/installing-platform-agnostic

https://68102--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks

https://68102--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-cluster-rhel-compute

https://68102--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes

QE review:
- [x] QE has approved this change.

Additional information:
4.11: https://github.com/openshift/openshift-docs/pull/68009
Main: https://github.com/openshift/openshift-docs/pull/67954

### CODE REVIEW

**Review by lahinson — COMMENTED** [review]
@JoeAldinger I made a couple of suggestions, but LGTM overall.

File: modules/installation-machine-requirements.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ endif::ibm-z[]
 ====
 
 ifndef::ibm-z,ibm-power[]
-The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.4, or {op-system-base} 8.5.
+The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.4–{op-system-base} 8.8.
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply modules/installation-machine-requirements.adoc:90]: cc @mjpytlak does that need an `and`?

[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/installation-machine-requirements.adoc:90]: It needs the "and." The control plane nodes can be either RHCOS or supported RHEL versions.

[the remark this answers is not in the document]
Comment (sdodson) [reply modules/installation-machine-requirements.adoc]: > The control plane nodes can be either RHCOS or supported RHEL versions.

I think you probably meant compute here, control plane nodes must be RHCOS. compute are the only place we allow RHEL OS.

**Pull request merged**